ros_buildfarm/templates/snippet/publisher_warnings.xml.em

```diff
@@ -19,7 +19,6 @@
       <sourceDirectory></sourceDirectory>
       <ignoreQualityGate>false</ignoreQualityGate>
       <ignoreFailedBuilds>true</ignoreFailedBuilds>
-      <referenceJobName>-</referenceJobName>
```

Comment:
* warnings-ng no longer uses a referenceJobName field and now has a more
complex/robust method for selecting a reference build using the Git
Forensics plugin
jenkinsci/warnings-ng-plugin@master/doc/Documentation.md#configure-the-selection-of-the-reference-build-baseline